packages/devtools/lib/src/debugger/debugger.dart

```diff
@@ -416,6 +464,8 @@ class DebuggerScreen extends Screen {
       scriptCountDiv.text = scriptsView.items.length.toString();
     });
 
+    scriptsView.onKeyDown.listen((html.KeyboardEvent e) {});
```

Comment:
remove this no-op line.

Reply:
> Strange. Typing "home." in the search box results in zero matches.
> typing "home.d" results in a match on "home.dart"

Was using RegEx removed, now using simple string comparison fixes the problem.

Reply:
Done and removed the onKeyDown debugging noop.